src/e2e-test/features/gcs/source/GCSSourceToBigQuery.feature

```diff
@@ -285,3 +285,304 @@ Feature: GCS source - Verification of GCS to BQ successful data transfer
     Then Verify the pipeline status is "Succeeded"
     Then Get count of no of records transferred to target BigQuery Table
     Then Validate the values of records transferred from GCS bucket file is equal to the values of target BigQuery table
+
+    # New test scenarios added by Bijay on Nov 13
+  @GCS_TSV_TEST @BQ_SINK_TEST
+  Scenario: To verify Successful GCS to BigQuery file format selection using tsv
+    Given Open Datafusion Project to configure pipeline
+    When Select plugin: "GCS" from the plugins list as: "Source"
+    When Sink is BigQuery
+    Then Connect source as "GCS" and sink as "BigQuery" to establish connection
+    Then Open GCS source properties
+    Then Enter GCS property projectId and reference name
+    Then Override Service account details if set in environment variables
+    Then Enter GCS source property path "gcsTsvFile"
+    Then Select GCS property format "tsv"
+    Then Toggle GCS source property skip header to true
+    Then Validate output schema with expectedSchema "gcsTsvFileSchema"
+    Then Validate "GCS" plugin properties
+    Then Close the GCS properties
+    Then Open BigQuery sink properties
+    Then Override Service account details if set in environment variables
+    Then Enter the BigQuery sink mandatory properties
+    Then Validate "BigQuery" plugin properties
+    Then Close the BigQuery properties
+    Then Save the pipeline
+    Then Preview and run the pipeline
+    Then Wait till pipeline preview is in running state
+    Then Open and capture pipeline preview logs
+    Then Verify the preview run status of pipeline in the logs is "succeeded"
+    Then Close the pipeline logs
+    Then Click on preview data for BigQuery sink
+    Then Verify preview output schema matches the outputSchema captured in properties
+    Then Close the preview data
+    Then Deploy the pipeline
+    Then Run the Pipeline in Runtime
+    Then Wait till pipeline is in running state
+    Then Open and capture logs
+    Then Verify the pipeline status is "Succeeded"
+    Then Get count of no of records transferred to target BigQuery Table
```

Comment:
Add the validation step at the end of each scenario--> Validate the values of records transferred from GCS bucket file is equal to the values of target BigQuery table

Reply:
Added.